challenges/profile_card_component/index.html

```diff
@@ -28,35 +28,85 @@
     >
-      <div>
-        <a
-          class="text-optimum-blue hover:text-optimum-darkblue"
-          href="https://www.frontendmentor.io/challenges/profile-card-component-cfArpWshJ"
-          target="_blank"
-          >Frontend Mentor challenge</a
-        >
-      </div>
-      <div>
-        for
-        <a
-          class="text-optimum-blue hover:text-optimum-darkblue"
-          href="https://github.com/optimumBA"
-          target="_blank"
-          >Optimum BH</a
+      <div class="m-6 max-w-sm rounded-2xl bg-white">
+        <div>
+          <img
+            class="block h-40 w-full rounded-t-2xl bg-no-repeat object-cover object-center"
+            src="../images/bg-pattern-card.svg"
+            alt=""
+          />
+        </div>
+
+        <img
+          class="relative m-auto mt-[-50px] block rounded-full border-[5px] border-white"
+          src="../images/image-victor.jpg"
+          alt=""
+        />
+        <h2 class="mt-4 text-xl font-bold text-gray-700">
+          Victor Crest <span class="font-normal text-gray-500">26</span>
+        </h2>
+        <p class="mt-2 text-lg text-gray-500">London</p>
+        <div
+          class="mt-8 flex items-center justify-around border-t-2 border-black border-opacity-10 px-6 py-6"
         >
-        Internship.
+          <div>
+            <h3 class="text-xl font-bold text-gray-800">80K</h3>
+            <p class="text-xs text-gray-700">Followers</p>
+          </div>
+          <div>
+            <h3 class="text-xl font-bold text-gray-800">803K</h3>
+            <p class="text-xs text-gray-700">Likes</p>
+          </div>
+          <div>
+            <h3 class="text-xl font-bold text-gray-800">1.4K</h3>
+            <p class="text-xs text-gray-700">Photos</p>
+          </div>
+        </div>
       </div>
+      <div class="credits rounded-2xl bg-white p-4">
+        <div>
+          <a
+            class="text-optimum-blue hover:text-optimum-darkblue"
+            href="https://www.frontendmentor.io/challenges/profile-card-component-cfArpWshJ"
+            target="_blank"
+            >Frontend Mentor challenge</a
+          >
+        </div>
+        <div>
+          for
+          <a
+            class="text-optimum-blue hover:text-optimum-darkblue"
+            href="https://github.com/optimumBA"
+            target="_blank"
+            >Optimum BH</a
+          >
+          Internship.
+        </div>
 
-      <div class="mt-3">
-        <a
-          class="text-optimum-blue hover:text-optimum-darkblue"
-          href="https://github.com/optimumBA/frontend_mentor_challenges/tree/profile_card_component/challenges/profile_card_component"
-          >Source code</a
-        >
+        <div class="mt-3">
+          <a
+            class="text-optimum-blue hover:text-optimum-darkblue"
+            href="https://github.com/optimumBA/frontend_mentor_challenges/tree/profile_card_component/challenges/profile_card_component"
```

Comment:
Like in the other PRs, put your repo here.

Reply:
Done :+1: , Thank you for the review @almirsarajcic